core/src/lib.rs

```diff
@@ -12,7 +12,7 @@ pub mod ledger;
 pub mod proto;
 pub mod types;
 
-#[cfg(feature = "abciplus")]
+#[cfg(not(feature = "abcipp"))]
```

Comment:
Had to change this otherwise `make check-abcipp` stopped working

Reply:
for the sake of consistency, I would even rename every feature flag to `#[cfg(not(feature = "abcipp"))]`. obviously that's out of the scope of this PR, though